dask/dataframe/__init__.py

```diff
@@ -33,11 +33,25 @@ def _dask_expr_enabled() -> bool:
         merge,
         pivot_table,
         read_csv,
+        read_hdf,
```

Comment:
tests are failing:

FAILED dask/array/tests/test_array_core.py::test_map_blocks_series - ImportError: cannot import name 'read_hdf' from 'dask_expr' (/home/runner/miniconda3/envs/test-environment/lib/python3.12/site-packages/dask_expr/__init__.py). Did you mean: 'read_csv'?
FAILED dask/tests/test_distributed.py::test_to_hdf_distributed - ImportError: cannot import name 'read_hdf' from 'dask_expr' (/home/runner/miniconda3/envs/test-environment/lib/python3.12/site-packages/dask_expr/__init__.py). Did you mean: 'read_csv'?
FAILED dask/tests/test_distributed.py::test_to_hdf_scheduler_distributed[1] - ImportError: cannot import name 'read_hdf' from 'dask_expr' (/home/runner/miniconda3/envs/test-environment/lib/python3.12/site-packages/dask_expr/__init__.py). Did you mean: 'read_csv'?
FAILED dask/tests/test_order.py::test_array_vs_dataframe[True] - ImportError: cannot import name 'read_hdf' from 'dask_expr' (/home/runner/miniconda3/envs/test-environment/lib/python3.12/site-packages/dask_expr/__init__.py). Did you mean: 'read_csv'?
FAILED dask/tests/test_order.py::test_array_vs_dataframe[False] - ImportError: cannot import name 'read_hdf' from 'dask_expr' (/home/runner/miniconda3/envs/test-environment/lib/python3.12/site-packages/dask_expr/__init__.py). Did you mean: 'read_csv'?

ERROR collecting dask/dataframe/io/tests/test_demo.py
  E   ImportError: cannot import name 'read_hdf' from 'dask_expr' (/home/runner/miniconda3/envs/test-environment/lib/python3.12/site-packages/dask_expr/__init__.py). Did you mean: 'read_csv'?
```

Reply:
This must be because of cache issues, I've merged all he io connectors yesterday evening in dask-expr